### PR TITLE
clock project completed

### PR DESCRIPTION
**Pull request by reihanehPourkaka**

### CODE REVIEW

**Comment by yeganemehr** [comment]
Hi Reihane
Thank you for participating in the challenge!
Your code is very clean and straightforward and very to close that kind of code I want to see everyday.
But can you remove `.idea` directory in root of repository so I can merge it?
Also I have some question that ask them separately.

File: challenges/2/YOUR-CODE-GOES-HERE/reihanehPourkaka/challenge/src/App.vue [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<script setup>
+import Clock from './components/Clock.vue'
+
+
+
+</script>
+
+<template>
+  <div class="container">
+    <Clock :activeHours='[{start: 5, end: 13}, {start: 22, end: 24}]'/>
```
Comment (yeganemehr) [inline challenges/2/YOUR-CODE-GOES-HERE/reihanehPourkaka/challenge/src/App.vue]: When I change these values (5, 13, 22, 24) component does not re-render.
Can you say why? and fix it?

File: challenges/2/YOUR-CODE-GOES-HERE/reihanehPourkaka/challenge/src/components/Clock.vue [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+checkValid();
+
+</script>
+
+<template>
+
+    <div class="timeRange"
+         v-for="(activeHour,index) in timeRange"
+         :key="index"
+         :style="{backgroundImage : 'conic-gradient(rgb(255 255 255 / 0%)  00deg  , rgb(255 255 255 / 0%) '+activeHour.start *15+'deg ,#ee7e25  '+activeHour.start *15+'deg  , #ee7e25  '+activeHour.end *15+'deg,rgb(255 255 255 / 0%)'+activeHour.end *15+'deg  , rgb(255 255 255 / 0%) 360deg)' }"
```
Comment (yeganemehr) [inline challenges/2/YOUR-CODE-GOES-HERE/reihanehPourkaka/challenge/src/components/Clock.vue]: What do first and last arguments of `conic-gradient` do? Nothing changed when I deleted them.

**Review by yeganemehr — CHANGES_REQUESTED** [review]

File: challenges/2/YOUR-CODE-GOES-HERE/reihanehPourkaka/challenge/src/App.vue (hunk 1 above)
Comment (reihanehPourkaka) [reply challenges/2/YOUR-CODE-GOES-HERE/reihanehPourkaka/challenge/src/App.vue]: hi
Yes , i forgot add watch .
i fixed it.

**Comment by yeganemehr** [comment]
I closing this due to inactivity.

**Pull request closed without merging**